tests/unit-js/src/test/scala/tests/js/JsSuite.scala

```diff
@@ -171,7 +200,15 @@ class JsSuite extends BaseMarkdownSuite {
        | required: String
        |val x: String = 42
        |                ^^
-    """.stripMargin
+    """.stripMargin,
+    compat = Map(
+      "3.0" ->
+        """
+          |-error: compile-only-error.md:3:17:
```

Comment:
Hmm. This is not right.

Reply:
Right. Seems like it's because the scala binary version has changed for 3.0.0 stable - so our tests aren't actually verifying Scala 3 assertions :-/